client-src/default/index.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+// webpack@5 doesn't inject node polyfill automatically
+window.global = window.global || window;
+window.process = window.process || {};
+window.process.env = window.process.env || {};
```

Comment:
@evilebottnawi I substituted `process/browser` out for this, I think it is all that is needed. I think we still need the `util/util` package as a dependency, since something in tapable needs it.